src/amberscript/parser.h

```diff
@@ -67,6 +67,9 @@ class Parser : public amber::Parser {
   Result ParseExpect();
   Result ParseCopy();
   Result ParseDeviceFeature();
+  Result ParseRepeat();
+  bool isRepeatable(const std::string& name) const;
```

Comment:
nit: Is there a reason this is lower-case-first?

Reply:
fix